File: app/src/main/java/com/example/james/rms/Core/ServePath/InventoryServePath.java
package com.example.james.rms.Core.ServePath;

import com.example.james.rms.NetWork.ServeProfile;

/**
 * Created by jamie on 2017/5/1.
 */

public class InventoryServePath {

    public static String serve_findByPartyIdAndStatus(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getInventorySum_findByPartyIdAndStatus();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String serve_findByPartyIdAndStatusOrderByProductId(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getInventorySum_findByPartyIdAndStatusOrderByProductId();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String serve_findByPartyIdAndStatusOrderByProductIdAsc(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getInventory_findByPartyIdAndStatusOrderByProductIdAsc();
        String serve_path = serve+path;
        return serve_path;
    }


}

File: app/src/main/java/com/example/james/rms/Core/ServePath/InventoryServePath.kt
package com.example.james.rms.Core.ServePath

import com.example.james.rms.NetWork.ServeProfile

/**
 * Created by jamie on 2017/5/1.
 */

object InventoryServePath {
    val serve: String
        get() = ServeProfile.serve

    fun serve_findByPartyIdAndStatus(): String {
        val path = ServeProfile.inventorySum_findByPartyIdAndStatus
        return serve + path
    }

    fun serve_findByPartyIdAndStatusOrderByProductId(): String {
        val path = ServeProfile.inventorySum_findByPartyIdAndStatusOrderByProductId
        return serve + path
    }

    fun serve_findByPartyIdAndStatusOrderByProductIdAsc(): String {
        val path = ServeProfile.inventory_findByPartyIdAndStatusOrderByProductIdAsc
        return serve + path
    }


}
